cloudflare_worker/README.md

```diff
@@ -29,24 +29,21 @@ limitations under the License.
    ```
    All following steps in this `README.md` should be done in this folder.
 
-1. Create a `wrangler.toml` from the template `wrangler.example.toml`.
-   1. Set `zone_id` and `account_id` from the values [pictured
-      here](https://forum.aapanel.com/d/3914-how-to-get-zone-id-of-cloudflare).
-   1. Change the domain in `routes` from `my_domain.com` to your domain.
-      ([Details
-      here.](https://developers.cloudflare.com/workers/get-started/guide#optional-configure-for-deploying-to-a-registered-domain))
-   1. Use the command `wrangler kv:namespace create OCSP` to create the id of
-      [KV namespace](https://developers.cloudflare.com/workers/runtime-apis/kv),
-      and put it into the `id` field in `kv_namespaces`. (If already created,
-      use the command `wrangler kv:namespace list` to get the id.)
-1. Create a `config.yaml` from the template `config.example.yaml`.
-   1. Put your domain as `html_host`.
-   1. In `worker_host`, replace `my_worker_subdomain` with the value from the
+1. Run `cargo run -p config-generator` and follow the command line prompt.
```

Comment:
Mention that this creates a `wrangler.toml` file that can be modified further if desired. (Or mention it in the Maintenance section.)

Reply:
Done